Portfolio.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE = html>
+<html lang="en">
+<head>
+  	<meta charset="utf-8">
+  	<title>Jenny Allton-Twist</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="Portfoliostyle.css" type="text/css" rel="stylesheet">	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+
+</head>
+
+<body>
+	<nav>
+		<ul>
+			<li><a href="#home">Home</a></li>
+			<li><a href="#portfolio">Portfolio</a></li>
+			<li><a href="#aboutme">About Me</a></li>
+			<li><a href="#contact">Contact</a></li>
+		 </ul>
+	</nav>
+
+
+
+	<div class="intropage" id="home">
+		<div class="container">
+			<h1>JENNY ALLTON-TWI<b>S</b>T</h1>
+		</div>
+	</div>
+
+
+
+	<div class="portfolio" id="portfolio"><h2>Portfolio</h2>
+		<div class="container">
+			<div class="box">
+				<a href="#"><img src="tech.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>October 2020, Mayden Academy, Bath UK</h4>
+			</div>
+			<div class="box">
+				<a href="#"><img src="woman.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>October 2020, Mayden Academy, Bath UK</h4>
+			</div>
+			<div class="box">
+				<a href="#"><img src="hacker.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>November 2020, Mayden Academy, Bath UK</h4>
+			</div>
+			<div class="box">
+				<a href="#"><img src="handshake.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>December 2020, Mayden Academy, Bath UK</h4>
+			</div>
+			<div class="box">
+				<a href="#"><img src="lightbulb.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>January 2021, Bristol UK</h4>
+			</div>
+			<div class="box">
```

Comment:
I'd use a different background colour on your box class to offset it against the background, also I'd consider renaming this class as box seems a bit generic considering it's nested within a specific section.

Reply:
background edited; I'm leaving box for now!

Reply:
Coolio!